autoctt_client.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+import getopt
+import socket
+import sys
+import cbor
+import json
+import time
+import traceback
+import subprocess
+
+from coapthon.client.helperclient import HelperClient
+from coapthon.utils import parse_uri
+from coapthon import defines
+from coapthon.serializer import Serializer
+
+__author__ = 'Giacomo Tanganelli'
```

Comment:
Who is Giacomo? 😛

Reply:
this is a new file.... so Frank can be the author..

Reply:
autoctt_client.py imports a lot of scripts and functions from this CoAPthon3 library so I thought it'd be easier to save it in this repository.

Reply:
> Who is Giacomo? 😛

Original author of this library. Part of autoctt_client.py are copied from his client scripts so I decided to leave his name there.

Reply:
> 
> 
> autoctt_client.py imports a lot of scripts and functions from this CoAPthon3 library so I thought it'd be easier to save it in this repository.

It doesn't actually need most of these imports, so they can be removed. This file in particular is very specific to our autoctt, so should almost certainly be in that repo, importing this one.